app-ios/Modules/Sources/Assets/Resources/Strings/en.lproj/Localizable.strings

```diff
@@ -16,6 +16,13 @@
 "about.app_version" = "App Version";
 "about.android_robot_description" = "The Android robot is reproduced or modified from work created and shared by Google and used according to terms described in the Creative Commons 3.0 Attribution License.";
 
+"search.day" = "Day";
+"search.category" = "Category";
+"search.session_type" = "Session type";
+"search.supported_languages" = "Support languages";
```

Comment:
It's a nits, but please fix it to "Supported languages" 🙏

Reply:
Oh.

Reply:
I checked your re-fix, thanks!